Netflix/metaflow-ui
# Bug/handle null event values

### Requirements for a pull request


### Description of the Change

A user reported that a metadata value containing the following data broke the UI

```
"field_name": "execution-triggers",
"value": "[{\"timestamp\": null, \"id\": null, \"name\": null, \"type\": \"event\"}]",
```

This change handles those `null` values more robustly.

### Alternate Designs

\-

### Possible Drawbacks

`null` values in the event are probably masking another underlying problem. Handling them gracefully may not alert the user that there is a problem.

### Verification Process

* Send a trigger with a `null` id and `null` name
* Confirm that the UI does not break and shows `unknown` for the event name

### Release Notes

Handle null values in execution trigger events more robustly.
